### PR TITLE
Add documentation comment for getRandomFruit()

### LINKED ISSUE #1: Write a documentation comment for `getRandomFruit()` method
Write a documentation comment for `getRandomFruit()` method in `MainActivity.kt` class.

To learn more about how to write a doc comment in Kotlin language, click [this link](https://kotlinlang.org/docs/coding-conventions.html#documentation-comments)

### PR DESCRIPTION
**Pull request by yagmurerdogan**

### CODE REVIEW

**Comment by AidaIssayeva** [comment]
Hi @yagmurerdogan 👋 ! Thank you so much for your contribution!

I'm currently away from my laptop - I'll review the PR during the day.

File: app/src/main/java/com/cupsofcode/basic/MainActivity.kt [hunk 1]
```diff
@@ -24,6 +24,10 @@ class MainActivity : AppCompatActivity() {
         "Blueberry"
     )
 
+    /**
+     * Returns random fruit from fruits as String
```
Comment (AidaIssayeva) [inline app/src/main/java/com/cupsofcode/basic/MainActivity.kt]: there should be a period at the end of the sentence.

File: app/src/main/java/com/cupsofcode/basic/MainActivity.kt [hunk 2]
```diff
@@ -24,6 +24,10 @@ class MainActivity : AppCompatActivity() {
...
+     */
+
```
Comment (AidaIssayeva) [inline app/src/main/java/com/cupsofcode/basic/MainActivity.kt]: no blank line between fun name and doc comment is needed.

**Review by AidaIssayeva — COMMENTED** [review]
@yagmurerdogan great job on your first PR 🎉 👏 !

Left two tiny comments regarding the doc comment [convention in Kotlin](https://kotlinlang.org/docs/coding-conventions.html#documentation-comments).

**Comment by yagmurerdogan** [comment]
I just sent new commit and updated documentation, thank you 😊

**Review by AidaIssayeva — APPROVED** [review]

**Comment by AidaIssayeva** [comment]
@yagmurerdogan thank you so much for the PR 🙏 !

**Pull request merged**